No method name `e.isDuplicateColumnError`

## 🔙 Regression
I am getting this exception with is not making the app install
```
../../../../flutter/.pub-cache/hosted/pub.dartlang.org/flutter_cache_manager-2.0.0/lib/src/storage/cache_info_repositories/cache_object_provider.dart:59:18: Error: The method 'isDuplicateColumnError' isn't defined for the class 'DatabaseException'.
 - 'DatabaseException' is from 'package:sqflite_common/src/exception.dart' ('../../../../flutter/.pub-cache/hosted/pub.dartlang.org/sqflite_common-1.0.0/lib/src/exception.dart').
Try correcting the name to the name of an existing method, or defining a method named 'isDuplicateColumnError'.
          if (!e.isDuplicateColumnError(CacheObject.columnKey)) rethrow;
                 ^^^^^^^^^^^^^^^^^^^^^^                                 
../../../../flutter/.pub-cache/hosted/pub.dartlang.org/flutter_cache_manager-2.0.0/lib/src/storage/cache_info_repositories/cache_object_provider.dart:82:18: Error: The method 'isDuplicateColumnError' isn't defined for the class 'DatabaseException'.
 - 'DatabaseException' is from 'package:sqflite_common/src/exception.dart' ('../../../../flutter/.pub-cache/hosted/pub.dartlang.org/sqflite_common-1.0.0/lib/src/exception.dart').
Try correcting the name to the name of an existing method, or defining a method named 'isDuplicateColumnError'.
          if (!e.isDuplicateColumnError(CacheObject.columnLength)) rethrow;
                 ^^^^^^^^^^^^^^^^^^^^^^                                 
                                                                        
                                                                        
FAILURE: Build failed with an exception.                                
                                                                        
* Where:                                                                
Script '/Users/degreat/flutter/packages/flutter_tools/gradle/flutter.gradle' line: 904
                                                                        
* What went wrong:                                                      
Execution failed for task ':app:compileFlutterBuildDebug'.              
> Process 'command '/Users/degreat/flutter/bin/flutter'' finished with non-zero exit value 1
                                                                        
* Try:                                                                  
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
```

### Old (and correct) behavior
`1.2.2` App was able to install

### Current behavior
`2.0.0`Exception stops app from installing

### Reproduction steps
1. Run `flutter run`
Then exception gets throw

### Configuration

**Version:** 2.x

**Platform:** 
- [x] :iphone: iOS
- [x] :robot: Android

--- a/flutter_cache_manager/example/pubspec.yaml
+++ b/flutter_cache_manager/example/pubspec.yaml
@@ -4,7 +4,7 @@ description: A new Flutter project.
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.6.0 <3.0.0"
+  sdk: ">=2.10.2 <3.0.0"
 
 dependencies:
   flutter:

--- a/flutter_cache_manager/pubspec.yaml
+++ b/flutter_cache_manager/pubspec.yaml
@@ -13,7 +13,7 @@ dependencies:
   uuid:  ^2.0.2
   http: ^0.12.0+2
   path: ^1.6.4
-  sqflite: ^1.1.7+2
+  sqflite: ^1.3.2+1
   pedantic: ^1.8.0+1
   clock: ^1.0.1
   file: ">=5.1.0 <7.0.0"

